check-deps/Makefile

```diff
@@ -50,6 +51,8 @@ CFLAGS_have_libcapstone  = $(shell pkg-config --cflags capstone 2> /dev/null)
 LDFLAGS_have_libcapstone = $(shell pkg-config --libs   capstone 2> /dev/null)
 CFLAGS_have_libunwind  = $(shell pkg-config --cflags libunwind 2> /dev/null)
 LDFLAGS_have_libunwind = $(shell pkg-config --libs   libunwind 2> /dev/null)
+CFLAGS_have_liblttng = -I.
```

Comment:
Is this really needed?  I think it'll search the current directory for `#include "..."` by default.